ruby/vampires.rb

```diff
@@ -0,0 +1,62 @@
+#process multiple users
+
+puts "How many employees will be processed?"
+num = gets.chomp.to_i
+counter = 1
+
+#user input
+
+def determine_vampire
+puts "What is your name?"
+name = gets.chomp
+puts "How old are you?"
+age = gets.chomp.to_i 
+puts "What year were you born?"
+dob = gets.chomp.to_i
+puts "Our company cafeteria serves garlic bread. Should we order some for you? (y/n)"
+garlic = gets.chomp
+puts "Would you like to enroll in the company health insurance? (y/n)"
+enroll = gets.chomp
+puts "Please list your allergies."
+$allergy = gets.chomp
+
+#determine if vampire
+
+loop do 
+  puts "any other allergies? If not, please enter 'done'"
+  @allergy = gets.chomp
```

Comment:
Instance variables are great but only when used as an instance attribute of something.  Look into these more when we reach classes but leave them out of singular methods and driver code.